tasks/yum.yml

```diff
@@ -9,18 +9,29 @@
   with_items:
     - sudo
     - gcc
-    - sqlite
     - supervisor
     - nginx
     - openssl
     - openssl-devel
     - libffi-devel
     - python-devel
+    - redis
     - python-firewall
     - libsemanage-python
     - policycoreutils-python
     - redhat-rpm-config
 
+- name: Install spatialite
+  when: openwisp2_database.engine == "django.contrib.gis.db.backends.spatialite"
+  apt: name={{ item }} state=latest
+  notify: reload systemd
+  with_items:
+    - sqlite
+    - gdal
+    - libproj-devel
+    - libgeos-dev
```

Comment:
Didn't find this either. I'm not sure what to looking for.

Reply:
may be `geos-devel`?
https://stackoverflow.com/questions/42097501/centos-libgeos-repository-missing

Reply:
May be (i found `geos` in EPEL package but i'm not 100% sure it is the same)

Reply:
usually devel packages contain shared libraries that are used for more advanced stuff, the stuff this kind of application usually need